src/templates/profile.ts

```diff
@@ -15,3 +16,61 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
+"""
+usecase ${name} {
+  input {
+    "field title"
+    // foo is required nullable string
```

Comment:
I find it confusing both using the comment and description. I'd remove the description or perhaps swap it:

```suggestion
    // required nullable string
    "Description of the field foo"
```